Makefile

```diff
@@ -64,6 +64,10 @@ build:
 build-client:
 	cd client/cmd && $(GO) build -o "$(HUSKYCICLIENTBIN)" && mv "$(HUSKYCICLIENTBIN)" ../..
 
+## Builds client to the executable file huskyci-client
+build-client-linux:
```

Comment:
Using `make build` and trying to run it in a linux container returned some errors. Added `build-client-linux` to help people build a linux binary.